builder/xenserver/common/common_config.go

```diff
@@ -18,11 +18,12 @@ type CommonConfig struct {
 	Password string `mapstructure:"remote_password"`
 	HostIp   string `mapstructure:"remote_host"`
 
-	VMName        string   `mapstructure:"vm_name"`
-	VMDescription string   `mapstructure:"vm_description"`
-	SrName        string   `mapstructure:"sr_name"`
-	FloppyFiles   []string `mapstructure:"floppy_files"`
-	NetworkNames  []string `mapstructure:"network_names"`
+	VMName            string   `mapstructure:"vm_name"`
+	VMDescription     string   `mapstructure:"vm_description"`
+	SrName            string   `mapstructure:"sr_name"`
+	FloppyFiles       []string `mapstructure:"floppy_files"`
+	NetworkNames      []string `mapstructure:"network_names"`
+	FinalNetworkNames []string `mapstructure:"final_network_names"`
```

Comment:
Could we call this `ExportNetworkNames` to make it clear that these will be used on Export? (Final depends on what operations packer is actually doing - sometimes it might not export the VM).